org.eclipse.scanning.server/src/org/eclipse/scanning/server/servlet/AcquireRequestHandler.java

```diff
@@ -110,28 +151,28 @@ private String getOutputFilePath(AcquireRequest request) throws EventException {
 				throw new EventException(e);
 			}
 		}
-		
+
 		return request.getFilePath();
 	}
-	
+
 	@SuppressWarnings("unchecked")
 	private void configureDetector(IRunnableDevice<?> detector, Object detectorModel,
-			ScanModel scanModel) throws Exception {
-		
+			ScanModel scanModel, IPointGenerator<?> gen) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)